src/SMTP.php

```diff
@@ -280,6 +280,8 @@ protected function edebug($str, $level = 0)
         }
         //Is this a PSR-3 logger?
         if ($this->Debugoutput instanceof \Psr\Log\LoggerInterface) {
+            //Remove ending line breaks potentialy added by calls to SMTP::client_send()
+            $str = preg_replace('/[' . static::LE .']+$/', '', $str);
```

Comment:
Instead of that `static::LE`, this should use an explicit `\r\n` as we are matching individual meta characters in a character class, not a literal string. `static::LE` contains a literal `"\r\n"` which represents actual CR and LF chars, not their meta-equivalents. Basically do what the other debug output options do.

Reply:
This would also fail the code style check, which requires spaces either side of string concat `.` chars, but that's academic as you won't need to do concat with the above change.